klaytn/src/adapter.ts

```diff
@@ -0,0 +1,98 @@
+import { Requester, Validator } from '@chainlink/external-adapter'
+import { ExecuteWithConfig, ExecuteFactory } from '@chainlink/types'
+import { logger } from '@chainlink/external-adapter'
+import { CustomConfig, makeConfig } from './config'
+import fs from 'fs'
+import Caver from 'caver-js'
+
+const makeTx = async (caver: any, keyring: any, input: any): Promise<txResult> => {
+  let { value } = input
+  const { address, data, topics } = input
+  value = JSON.parse(value).result
```

Comment:
@justinkaseman This line doesn't parse the input.
```suggestion
  const address = input.data.address
  const data = input.data.data
  const topics = input.data.topics
  let value = JSON.parse(input.data.value).result
```

Reply:
Oops, good catch :sweat_smile: